Monika After Story/game/zz_apikeys.rpy

```diff
@@ -439,14 +439,14 @@ init -980 python in mas_api_keys:
             # type check
             try:
                 if not isinstance(rv, tuple):
-                    raise TypeError(ERR_ON_CHG_TYPE_NOT_TUPLE.format(rv))
+                    raise TypeError(ERR_ON_CHG_TYPE_NOT_TUPLE.format(type(rv)))
 
                 if len(rv) < 2:
                     raise TypeError(ERR_ON_CHG_TYPE_BAD_TUP_SIZE.format(len(rv)))
 
                 # only check error message if on_change is returning false
-                if not rv[0] and not isinstance(rv[1], (str, unicode)):
-                    raise TypeError(ERR_ON_CHG_TYPE_BAD_ERR_MSG.format(rv[1]))
+                if not rv[0] and not isinstance(rv[1], basestring):
```

Comment:
Can we add a py3 TODO on this, basestring doesn't exist in py3

Reply:
This is my bad actually